custom_model_runner/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+#### [1.9.2] - 2022-04-01
+##### Fixed
+- Unpin werkzeug
```

Comment:
I think this changelog note should be updated

Reply:
DONE